gcp-observability/src/main/java/io/grpc/gcp/observability/ObservabilityConfigImpl.java

```diff
@@ -74,95 +87,128 @@ void parse(String config) throws IOException {
 
   private void parseConfig(Map<String, ?> config) {
     if (config != null) {
-      Boolean value = JsonUtil.getBoolean(config, "enable_cloud_logging");
-      if (value != null) {
-        enableCloudLogging = value;
-      }
-      value = JsonUtil.getBoolean(config, "enable_cloud_monitoring");
-      if (value != null) {
-        enableCloudMonitoring = value;
-      }
-      value = JsonUtil.getBoolean(config, "enable_cloud_trace");
-      if (value != null) {
-        enableCloudTracing = value;
-      }
-      destinationProjectId = JsonUtil.getString(config, "destination_project_id");
-      flushMessageCount = JsonUtil.getNumberAsLong(config, "flush_message_count");
-      List<?> rawList = JsonUtil.getList(config, "log_filters");
-      if (rawList != null) {
-        List<Map<String, ?>> jsonLogFilters = JsonUtil.checkObjectList(rawList);
-        ImmutableList.Builder<LogFilter> logFiltersBuilder = new ImmutableList.Builder<>();
-        for (Map<String, ?> jsonLogFilter : jsonLogFilters) {
-          logFiltersBuilder.add(parseJsonLogFilter(jsonLogFilter));
-        }
-        this.logFilters = logFiltersBuilder.build();
+      projectId = JsonUtil.getString(config, "project_id");
+      parseLoggingObject(config, CONFIG_LOGGING_VAR_NAME);
```

Comment:
nit: Is there a reason the literal strings weren't used here? I don't see what is special about these to have made them into constants.

Reply:
Since these are top level objects we look for in the config, I added them as constants.

Reply:
`"project_id"` is also a "top level object" but we didn't use a constant so I agree with Eric

Reply:
Removed all constant added specific to config.